ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -53,6 +53,8 @@ presubmits:
           sources:
           - secret:
               name: cluster-secrets-aws
+          - configMap:
```

Comment:
I'm not clear on what the issue is, but [CI doesn't like these][1].  Maybe try again with them before the `- secret` block to keep the entries alphabetized?

[1]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_release/2353/pull-ci-openshift-release-master-generated-config/1584/build-log.txt

Reply:
It's not an ordering issue. These jobs are [Prowgen](https://github.com/openshift/ci-operator-prowgen)-generated and the failing `ci/prow/generated-config` presubmit is in place to keep these jobs from drifting away from the generated state.

So the question would be: is this something that *all* these jobs are supposed to have? If yes, Prowgen needs to learn it, somewhere around [here](https://github.com/openshift/ci-operator-prowgen/blob/master/cmd/ci-operator-prowgen/main.go#L162). If no, we'll need to hear some good reasons why this should be customizable.

Reply:
@petr-muller each of the end-to-end jobs need the install-config template, which varies by platform. I was hoping to just write those templates in an adjacent directory and plumb it through. What is the best way to achieve that here?

Reply:
We can revisit this later. We need this change in soon.